### DISCUSSION TITLE
High CPU usage

### DISCUSSION THREAD

**Discussion by dineshblusky** [supabase/supabase#30347] [Questions]
Hello, We are facing high CPU usage. Could you help us identifying how we could fix this issue? Any specific queries that helps us to address this issue?

**Comment by GaryAustin1** [comment]
You can look at 
![image](https://github.com/user-attachments/assets/3858d468-8c95-4ad2-bfb1-c6ac60ea42c1)
In the dashboard database tools to see if you have query issues.
You can look at the Gateway API Logs to see if you have more than normal requests coming in.
